test/mind_the_gapp/arrival_time_test.exs

```diff
@@ -0,0 +1,75 @@
+defmodule MindTheGapp.ArrivalTimeTest do
+  use MindTheGapp.DataCase, async: true
+  alias MindTheGapp.Mta.ArrivalTime
+
+  setup_all :create_fake_arrival_time_data
+
+  def create_fake_arrival_time_data(_context) do
+    clear_database()
+    generate_fake_arrival_data()
+    :ok
+  end
+
+  def clear_database() do
+    MindTheGapp.Repo.delete_all(MindTheGapp.ArrivalTime)
+  end
+
+  def generate_fake_arrival_data() do
+    starting_date = DateTime.utc_now
+
+     1..50
+     |> Enum.reduce(starting_date, &persist_arrival_time/2)
+  end
+
+  def persist_arrival_time(_x, date_time) do
+    actual_time = DateTime.add(date_time, Enum.random(-10..10) * 60)
+
+     %MindTheGapp.ArrivalTime{estimated: date_time, actual: actual_time}
+     |> MindTheGapp.Repo.insert!()
+
+    date_time |> DateTime.add(3600 * -6)
+  end
+  
+  describe "ArrivalTime" do
+    test "returns current day to seventh day before" do
+      query_results = ArrivalTime.difference_in_arrival_times
+      current_day = query_results |> hd |> hd
+      last_day = query_results |> Enum.take(-1) |> hd |> hd
```

Comment:
It would be more idiomatic to use pattern matching instead of calling `hd` multiple times.

```suggestion
      [current_day, _average] = List.first(result)
      [last_day, _average] = List.last(result)
```